explorer/src/components/MixnodesTable.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
```

Comment:
Do we need this component now that we have the data grid?

Reply:
Probably not, but i am still using it in the actual detail page still (right now)